### PR TITLE
[dhcp_sever] Add missing param in config cli

### PR DESCRIPTION
**Pull request by yaqiangz**
#### Why I did it
db param missed in dhcp_server config cli

##### Work item tracking
- Microsoft ADO **(number only)**: 27300333

#### How I did it
Add param

#### How to verify it
Build image, run cli

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205
- [ ] 202211
- [ ] 202305

#### Tested branch (Please provide the tested image version)

- [ ] 
- [ ]

### CODE REVIEW

File: rules/config [hunk 1]
```diff
@@ -150,7 +150,7 @@ INCLUDE_NAT = y
 INCLUDE_DHCP_RELAY = y
 
 # INCLUDE_DHCP_SERVER - build and install dhcp-server package
-INCLUDE_DHCP_SERVER ?= n
+INCLUDE_DHCP_SERVER ?= y
```
Comment (yaqiangz) [inline rules/config:153]: Remember to set this value to n when this pr is ready

**Review by Xichen96 — APPROVED** [review]

**Comment by yaqiangz** [comment]
@yxieca Could you please help to merge this PR?

**Comment by yxieca** [comment]
@yaqiangz how did this missed in the previous PR? Please add test to catch such issue.

**Comment by yaqiangz** [comment]
Fix included in https://github.com/sonic-net/sonic-buildimage/pull/18501

File: dockers/docker-dhcp-server/cli/config/plugins/dhcp_server.py [hunk 2]
```diff
@@ -47,7 +47,7 @@ def validate_str_type(type_, value):
 
 @click.group(cls=clicommon.AbbreviationGroup, name="dhcp_server")
 @clicommon.pass_db
-def dhcp_server():
+def dhcp_server(db):
```
Comment (w1nda) [inline dockers/docker-dhcp-server/cli/config/plugins/dhcp_server.py:50]: Hi @yaqiangz

**Pull request closed without merging**